tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -142,6 +147,10 @@ pub enum TrustedCall {
 	send_erroneous_parentchain_call(Identity),
 	#[codec(index = 25)]
 	maybe_create_id_graph(Identity, Identity),
+	#[codec(index = 25)]
+	update_token_staking_amounts(Identity, Vec<(AccountId, Balance)>),
+	#[codec(index = 27)]
+	reward_distribution_completed(Identity),
```

Comment:
Can these two calls be merged into one ?

We assume they are executed together and maybe there is strict ordering requirement ?

Reply:
After the discussions, we are going back to putting all the logic in the event_handler